extensions/vscode/lsp/client/tsconfig.json

```diff
@@ -0,0 +1,19 @@
+{

```

Comment:
I think you should probably reorganize the directory structure.
What is currently in `extensions/vscode/lsp` should probably just be: `extensions/vscode`, since it's not just an LSP extension, but the general-purpose vscode extension for wake. Conversely, since there is no `server`, only a `client`, I think the `extensions/vscode/lsp/client` directory should be `extensions/vscode/lsp-client`.

Reply:
Done.